src/configuration.ts

```diff
@@ -26,8 +24,8 @@ const knownInfuraNames = ['mainnet', 'ropsten', 'rinkeby', 'goerli', 'kovan', 'a
  * ```
  */
 export interface ProviderConfiguration extends Omit<EthrDidRegistryDeployment, 'chainId'> {
-  provider?: Provider
-  chainId?: string | number
+  provider?: Provider | null
+  chainId?: string | bigint
```

Comment:
Please revert to using numbers.

It's fine to accept bigint, but numbers must be accepted to allow configs to be serialized to plain config files.